scripts/simple.py

```diff
@@ -0,0 +1,7 @@
+def uftrace_entry(args):
```

Comment:
Trivial things, we don't need to add `#!/usr/bin/env python` at the very beginning line ?